completions/ip

```diff
@@ -181,13 +181,24 @@ _comp_cmd_ip()
         a | addr | address)
             case $subcmd in
                 add | change | replace)
-                    if [[ $prev == dev ]]; then
-                        _comp_compgen_available_interfaces
-                    elif [[ $prev == scope ]]; then
-                        _comp_cmd_ip__iproute2_etc rt_scopes
-                    else
-                        : # TODO
-                    fi
+                    case $prev in
+                        dev | label)
+                            _comp_compgen_available_interfaces
+                            ;;
+                        scope)
+                            _comp_cmd_ip__iproute2_etc rt_scopes
+                            ;;
+                        broadcast | anycast | peer | metric)
+                            :
+                            ;;
+                        valid_lft | preferred_lft)
+                            _comp_compgen -- -W 'forever'
+                            ;;
+                        *)
+                            _comp_compgen -- -W 'anycast broadcast dev home label scope nodad metric mngtmpaddr
+			        noprefixroute valid_lft peer preferred_lft'
```

Comment:
Ubuntu 24.04 has `autojoin` in `CONFFLAG`

Reply:
Added